cmd/aro/update_ocp_versions.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd"
+	"github.com/Azure/ARO-RP/pkg/util/encryption"
+	"github.com/Azure/ARO-RP/pkg/util/keyvault"
+)
+
+func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShiftVersion, error) {
+	env, err := env.NewCoreForCI(ctx, log)
+	if err != nil {
+		return nil, err
+	}
+	dstAcr := os.Getenv("DST_ACR_NAME")
+	acrDomainSuffix := "." + env.Environment().ContainerRegistryDNSSuffix
+
+	dstRepo := dstAcr + acrDomainSuffix
+	var (
+		OpenshiftVersions = []api.OpenShiftVersion{
+			{
+				Version:           "4.10.20",
+				OpenShiftPullspec: "quay.io/openshift-release-dev/ocp-release@sha256:e1fa1f513068082d97d78be643c369398b0e6820afab708d26acda2262940954",
```

Comment:
@darthhexx should we take this from version/const.go?

```
Version: version.InstallStream.Version.String(),
OpenShiftPullspec: version.InstallStream.PullSpec,
InstallerPullspec: fmt.Sprintf("%s/aro-installer:release-%s", dstRepo, version.InstallStream.Version.MinorVersion()),
```

Reply:
Good call, done.